packages/components/src/tour_tip/tour_tip.scss

```diff
@@ -1,3 +1,4 @@
+@import '../../../../sass/utils/mixins';
```

Comment:
not sure if we want to pull webapp files this way into the package components. However I am not sure if there is a way to add upside paths to the tsconfig.
cc: @hmhealey @M-ZubairAhmed

Reply:
That does seem like something we shouldn't be doing, but I don't know about Sass to know how it'll handle that, especially when we eventually get around to moving the web app code into a separate workspace. Doing this won't work for JS code, but maybe it'll work fine with CSS. Alternatively, we could consider moving these mixins into the components package and making them available for use outside of the package, but I don't know how that would be done in practice yet.

Anyway, I think that it's probably fine to do this if it works for now. At some point, we'll probably have to find a proper way to share these mixins between packages, but I don't think we need to worry about that yet

Reply:
ok! thanks